h2o-py/h2o/schemas/schema.py

```diff
@@ -33,14 +33,17 @@ def define_from_schema(cls, connection):
                 delattr(cls, attr)
         cls._schema_attrs_ = {}
 
-        for f in meta.fields:
-            name = f.name
-            if name not in cls._ignored_schema_keys_:
-                if hasattr(cls, name):
-                    cls._schema_attrs_[name] = False 
-                else:
-                    cls._schema_attrs_[name] = True
-                    setattr(cls, name, property(partial(cls.__getitem__, name=name), doc=f.help))
+        if hasattr(meta, "fields"):
+            for f in meta.fields:
+                name = f.name
+                if name not in cls._ignored_schema_keys_:
+                    if hasattr(cls, name):
+                        cls._schema_attrs_[name] = False 
+                    else:
+                        cls._schema_attrs_[name] = True
+                        setattr(cls, name, property(partial(cls.__getitem__, name=name), doc=f.help))
+        else:
+            raise H2OServerError("Unexpected API output. Please verify server url. Response is '%s'" % meta)
```

Comment:
LGTM, just a generic coding style suggestion to avoid deep indentations:
```python
if failure_cond:
    raise Error()
normal_behaviour:
    with:
        some:
            indent 

# nicer than
if success_cond:
    normal_behaviour:
        with:
            some:
                indent 
else:
    raise Error()
```

Reply:
Thank you, I changed it to your suggestion 👍

Reply:
except that you kept the unnecessary `else` ;)

Reply:
True, thank you!